misc/docs/source/tutorial/code_snippets/coordinate_conversions.rst

```diff
@@ -22,3 +22,21 @@ coordinates of two German stations to the regionally used Gauß-Krüger system:
 
 .. _`pyproj`: https://pypi.python.org/pypi/pyproj
 .. _`EPSG`: https://www.epsg-registry.org/
+
+
+Another common usage is to convert location information in latitude and
+longitude to `Universal Transverse Mercator coordinate system (UTM)`_. This is
+especially useful for large dense array in a small area. Such conversion can
```

Comment:
@ Thanks for the review. Yes, change from "array" to "arrays" makes much sense. English is not my first language. But isn't "usage" an uncountable norm? Anyway, I think the UTM is only a *single* usage of possible location conversion. Thus, I will keep the singular form.